src/extension.ts

```diff
@@ -69,6 +69,85 @@ import { registerCondaFeatures } from './managers/conda/main';
 import { registerPoetryFeatures } from './managers/poetry/main';
 import { registerPyenvFeatures } from './managers/pyenv/main';
 
+/**
+ * Collects relevant Python environment information for issue reporting
+ */
+async function collectEnvironmentInfo(
+    context: ExtensionContext,
+    envManagers: EnvironmentManagers,
+    projectManager: PythonProjectManager
+): Promise<string> {
+    const info: string[] = [];
+    
+    try {
+        // Extension version
+        const extensionVersion = context.extension?.packageJSON?.version || 'unknown';
+        info.push(`Extension Version: ${extensionVersion}`);
+        
+        // Python extension version
+        const pythonExtension = extensions.getExtension('ms-python.python');
```

Comment:
@karthiknadig ideas on other info we need that will be useful? Still need test everything it is getting here but wanted to get your thoughts on if there is any info I can add